tests/keras/engine/test_topology.py

```diff
@@ -76,6 +76,35 @@ def test_trainable_weights():
     assert model.non_trainable_weights == weights
 
 
+def test_compute_mask():
+    model = Sequential()
+    model.add(Dense(1, input_dim=2))
+    assert model.layers[0].supports_masking is True
+    assert model.layers[0].compute_mask([model.input], [0., 1.]) == [0., 1.]
+
+    model = Sequential()
+    model.add(Conv2D(1, [2, 2], input_shape=[3, 3, 1]))
+    assert model.layers[0].supports_masking is False
+    assert model.layers[0].compute_mask([model.input], [None]) is None
+
+    mask = np.array([[0., 1.], [1., 0.]])
+    with pytest.raises(TypeError):
+        model.layers[0].compute_mask([model.input], [mask])
+    with pytest.raises(TypeError):
+        model.layers[0].compute_mask([model.input], mask)
+
+
+def test_get_layer():
```

Comment:
It looks like this is testing only out-of-bounds layer requests. I think it would be worth adding the positive case in as well where you get a valid layer.